### PR TITLE
feat: prevent outbound network requests

### PR DESCRIPTION
**Pull request by SgtPooki**
- chore: add test fixtures
- feat: fix tests to prevent outbound network calls

Addresses some concerns from https://github.com/ipfs-shipyard/ipfs-geoip/pull/112#discussion_r1398054529 and prevents outbound network requests. We should add e2e tests for verifying that the network can supply the content for the expected CIDs, or have manual verification methods for that.

### CODE REVIEW

**Comment by SgtPooki** [comment]
this is currently failing on nodejs 20.9.0.. able to reproduce locally

File: test/lookupMultiple.node.spec.js [hunk 1]
```diff
@@ -1,30 +1,61 @@
...
+  let rewiredGeoIp
+  let failedCalls = 0
+
+  beforeEach(async () => {
+    failedCalls = 0
+  })
+  afterEach(() => {
+    rewiredGeoIp = null
+    failedCalls = 0
+  })
```
Comment (whizzzkid) [inline test/lookupMultiple.node.spec.js]: why both?

File: .eslintrc [hunk 2]
```diff
@@ -7,5 +7,6 @@
   "parserOptions": {
     "sourceType": "module",
     "allowImportExportEverywhere": true
-  }
+  },
+  "ignorePatterns": ["**/test/fixtures/**", "**/bin/load-fixtures.sh", "**/dist/**", "**/node_modules/**"],
```
Comment (whizzzkid) [inline .eslintrc]: optional

```suggestion
  "ignorePatterns": [
    "**/bin/load-fixtures.sh", 
    "**/dist/**", 
    "**/node_modules/**",
    "**/test/fixtures/**"
  ],
```

File: .aegir.js [hunk 3]
```diff
@@ -20,14 +23,40 @@ export default {
...
+      })
+      let gwUrl = process.env.IPFS_GATEWAY
+      if (!gwUrl) {
+        // no gateway specified, start the server
+        await new Promise((resolve, _reject) => {
+          server.listen(0, () => {
+            gwUrl = `http://localhost:${server.address().port}`
+            console.log(`server listening at ${gwUrl}`)
+            resolve()
+          })
```
Comment (whizzzkid) [inline .aegir.js:47]: I'm forgetting this, but isn't the first param the port? then host, then callback?

**Review by whizzzkid — APPROVED** [review]
lgtm, minor nits.

File: .aegir.js (hunk 3 above)
Comment (SgtPooki) [reply .aegir.js:47]: first param is port, 0 picks an open one on localhost. see https://nodejs.org/api/net.html#serverlistenport-host-backlog-callback

**Review by lidel — APPROVED** [review]
:ok_hand:

**Pull request merged**